### ISSUE TITLE
Using <body hidden aria-hidden="true">

### ISSUE THREAD

**Issue by design1online** [w3c/aria#1458]
Should we make it explicit that we cannot use aria-hidden on specific html elements?

Reference for this question is here: https://www.matuzo.at/blog/building-the-most-inaccessible-site-possible-with-a-perfect-lighthouse-score/

**Comment by aleventhal** [comment]
This came up before in another issue somewhere. I think aria-hidden is legitimate on an <html> or <body> if it's inside of an iframe, but should be ignored by browser agents if it's in the top level document.

**Comment by jnurthen** [comment]
I think this is a duplicate of #1254 . @design1online please close if you agree.

**Comment by scottaohara** [comment]
beat me to it, james.  

as I mentioned on the call today, in regards to disallowing `aria-hidden=true` on the `body` element, specifically, it would make sense to update [ARIA in HTML](https://github.com/w3c/html-aria/issues/221).

**Comment by design1online** [comment]
Yep it's a duplicate, thanks. Closing.
